test/e2e/pvutil.go

```diff
@@ -130,35 +131,35 @@ func deletePVCandValidatePV(c clientset.Interface, ns string, pvc *v1.Persistent
 	pvc, err := c.Core().PersistentVolumeClaims(ns).Get(pvc.Name, metav1.GetOptions{})
 	Expect(apierrs.IsNotFound(err)).To(BeTrue())
 
-	// Wait for the PV's phase to return to Available
+	// Wait for the PV's phase to return to be `expectPVPhase`
 	framework.Logf("Waiting for recycling process to complete.")
-	err = framework.WaitForPersistentVolumePhase(expctPVPhase, c, pv.Name, 1*time.Second, 300*time.Second)
+	err = framework.WaitForPersistentVolumePhase(expectPVPhase, c, pv.Name, 1*time.Second, 300*time.Second)
 	Expect(err).NotTo(HaveOccurred())
 
 	// examine the pv's ClaimRef and UID and compare to expected values
 	pv, err = c.Core().PersistentVolumes().Get(pv.Name, metav1.GetOptions{})
 	Expect(err).NotTo(HaveOccurred())
 	cr := pv.Spec.ClaimRef
-	if expctPVPhase == v1.VolumeAvailable {
+	if expectPVPhase == v1.VolumeAvailable {
 		if cr != nil { // may be ok if cr != nil
 			Expect(len(cr.UID)).To(BeZero())
 		}
-	} else if expctPVPhase == v1.VolumeBound {
+	} else if expectPVPhase == v1.VolumeBound {
 		Expect(cr).NotTo(BeNil())
 		Expect(len(cr.UID)).NotTo(BeZero())
 	}
```

Comment:
when released, no claimRef fields are altered so the check wouldn't find anything interesting.